server/channels/web/oauth.go

```diff
@@ -76,7 +76,7 @@ func authorizeOAuthApp(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	auditRec.Success()
-	c.LogAudit("")
+	c.LogAudit("success")
```

Comment:
Unify style

Reply:
This new "style" is more error prone in that "success" can be misspelled.  Also the linter will eventually complain that the "success" string should be a constant.

Reply:
Why do we need two ways of marking an audit item as successful?

Reply:
As far as I understand, there are two types of audit events in Mattermost:
1. The newly (by you) added audit logs, which use the advanced logging feature to write to, e.g. a file. (Created via `c.MakeAuditRecord`)
2. The (old) audit records, which are written to the DB and served via http://localhost:8065/admin_console/compliance/monitoring. (Compliance Monitoring) (Created via `c.LogAudit("success")`)

Based on the wording in the system console, Compliance Monitoring seems to be removed in a future release. Hence, I'm hesitant to invest more time into 2.

Does that clear things up @wiggin77 ?